includes/admin/class-wc-rest-stripe-account-keys-controller.php

```diff
@@ -0,0 +1,208 @@
+<?php
+/**
+ * Class WC_REST_Stripe_Account_Keys_Controller
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * REST controller for saving Stripe's test/live account keys. This includes Live Publishable Key,
+ * Live Secret Key, Webhook Secret.
+ */
+class WC_REST_Stripe_Account_Keys_Controller extends WP_REST_Controller {
```

Comment:
If you extend `WC_Stripe_REST_Controller`, the class will inherit `check_permission()` and `$namespace` which can then be removed from this class:

```suggestion
class WC_REST_Stripe_Account_Keys_Controller extends WC_Stripe_REST_Controller {
```

Reply:
Good idea!

Reply:
Updated